src/game/client/components/stats.cpp

```diff
@@ -0,0 +1,314 @@
+#include <engine/shared/config.h>
+#include <engine/textrender.h>
+#include <engine/graphics.h>
+#include <engine/serverbrowser.h>
+#include <game/client/animstate.h>
+#include <game/client/components/sounds.h>
+#include <game/client/gameclient.h>
+#include <generated/client_data.h>
+#include "stats.h"
+
+CStats::CStats()
+{
+	m_Mode = 0;
+}
+
+void CStats::OnReset()
+{
+	for(int i=0; i<MAX_CLIENTS; i++)
+		m_pClient->m_aStats[i].Reset();
+	m_Mode = 0;
+}
+
+void CStats::ConKeyStats(IConsole::IResult *pResult, void *pUserData)
+{
+	if(pResult->GetInteger(0) != 0)
+		((CStats *)pUserData)->m_Mode = 1;
+	else
+		((CStats *)pUserData)->m_Mode = 0;
+}
+
+void CStats::OnConsoleInit()
+{
+	Console()->Register("+stats", "i", CFGFLAG_CLIENT, ConKeyStats, this, "Show stats");

```

Comment:
That's odd. action commands don't have a parameter.

Reply:
After `bind e +stats 1`, 

- pressing `e` sends `+stats 1`
- releasing `e` sends `+stats`

Apparently, this is used to close the statboard on release. Removing the parameter breaks it :/

Reply:
Replace the content of function ConKeyStats with:

`((CStats *)pUserData)->m_Active = pResult->GetInteger(0) != 0;`

and then remove the parameter from the comand. Works alright.

Reply:
Thanks, that was silly